supplements/sword-coast-adventurers-guide/deities.xml

```diff
@@ -0,0 +1,1510 @@
+		<description>
+			<p class="flavor">triple goddess of wisdom and protection</p>
+		</description>
+		<setters>
+			<set name="setting">Forgotten Realms</set>
+			<set name="alignment">CG</set>
+			<set name="domains">Knowledge, Life</set>
+			<set name="symbol">Triangle with three interlocking circles within</set>
+			<set name="gender">female</set>
+		</setters>
+		<sheet>
+			<description>triple goddess of wisdom and protection</description>
+		</sheet>
+	</element>
+	<element name="Corellon Larethian" type="Deity" source="Sword Coast Adventurer’s Guide" id="ID_WOTC_SCAG_DEITY_CORELLON_LARETHIAN">
+		<description>
+			<p class="flavor">god of art and magic</p>
+		</description>
+		<setters>
+			<set name="setting">Forgotten Realms</set>
+			<set name="alignment">CG</set>
+			<set name="domains">Arcana, Light</set>
+			<set name="symbol">Crescent moon</set>
+			<set name="gender">male</set>
+		</setters>
+		<sheet>
+			<description>god of art and magic</description>
+		</sheet>
+	</element>
+	<element name="Deep Sashelas" type="Deity" source="Sword Coast Adventurer’s Guide" id="ID_WOTC_SCAG_DEITY_DEEP_SASHELAS">
+		<description>
+			<p class="flavor">god of the sea</p>
+		</description>
+		<setters>
+			<set name="setting">Forgotten Realms</set>
+			<set name="alignment">CG</set>
+			<set name="domains">Nature, Tempest</set>
+			<set name="symbol">Dolphin</set>
+			<set name="gender">male</set>
+		</setters>
+		<sheet>
+			<description>god of the sea</description>
+		</sheet>
+	</element>
+	<element name="Erevan Ilesere" type="Deity" source="Sword Coast Adventurer’s Guide" id="ID_WOTC_SCAG_DEITY_EREVAN_ILESERE">
+		<description>
+			<p class="flavor">god of mischief</p>
+		</description>
+		<setters>
+			<set name="setting">Forgotten Realms</set>
+			<set name="alignment">CN</set>
+			<set name="domains">Trickery</set>
+			<set name="symbol">Asymmetrical eight-armed star</set>
+			<set name="gender">male</set>
+		</setters>
+		<sheet>
+			<description>god of mischief</description>
+		</sheet>
+	</element>
+	<element name="Fenmarel Mestarine" type="Deity" source="Sword Coast Adventurer’s Guide" id="ID_WOTC_SCAG_DEITY_FENMAREL_MESTARINE">
+		<description>
+			<p class="flavor">god of outcasts</p>
+		</description>
+		<setters>
+			<set name="setting">Forgotten Realms</set>
+			<set name="alignment">CN</set>
+			<set name="domains">Trickery</set>
+			<set name="symbol">Two peering elven eyes</set>
+			<set name="gender">male</set>
+		</setters>
+		<sheet>
+			<description>god of outcasts</description>
+		</sheet>
+	</element>
+	<element name="Hanali Celanil" type="Deity" source="Sword Coast Adventurer’s Guide" id="ID_WOTC_SCAG_DEITY_HANALI_CELANIL">
+		<description>
+			<p class="flavor">goddess of love and beauty</p>
+		</description>
+		<setters>
+			<set name="setting">Forgotten Realms</set>
+			<set name="alignment">CG</set>
+			<set name="domains">Life</set>
+			<set name="symbol">Golden Heart</set>
```

Comment:
typo, 'Heart' -> 'heart' (inkeeping with only first letter uppercase for other symbols)